DataCleaningScripts/update_portal_plots.R

```diff
@@ -0,0 +1,57 @@
+#' Appends new dates to Portal_plots
+#'
+#'
+#' 
+#' @example update_portal_plots()
+#' 
+#'
+#'
+library(zoo)
+update_portal_plots <- function() {
+#load plot data
+portal_plots = read.csv("../SiteandMethods/Portal_plots.csv")
+# load rodent data
+rodentdat = read.csv("../Rodents/Portal_rodent.csv",stringsAsFactors = FALSE,as.is=TRUE,na.strings = '')  
+
+# define currrent plot treatments
+
+controls = c(4,5,6,7,11,13,14,17,18,24)
+removals = c(1,9,10,12,16,23)
+exclosures = c(2,3,8,15,19,20,21,22)
+
+# proceed only if rodentdat has more recent data than plot data
+lastplotdates=max(as.yearmon(paste(substr(portal_plots$yr,3,4),portal_plots$month), "%y%m"))
+newdates=max(as.yearmon(paste(substr(rodentdat$yr,3,4),rodentdat$mo), "%y%m"))
+
+if (newdates > 
+    lastplotdates) {
+  
+  yr = rep(year(newdates),24)
+  month = rep(month(newdates),24)
+  plot = 1:24
```

Comment:
That's my next commit. Then I've just got one more for pulling the new moon dates from that url you linked to.